lib/xcp_service.ml

```diff
@@ -300,6 +300,9 @@ let startswith prefix x =
         prefix' <= x' && (String.sub x 0 prefix' = prefix)
 
 let configure_common ~options ~resources arg_parse_fn =
+  (* Register the Logs reporter to ensure we get log messages from libraris using Logs *)
+  Debug.init_logs ();
```

Comment:
Indentation

Reply:
I think this is the correct indentation, the rest of the function is incorrectly indented :D 
We should reindent this at some point

Reply:
I don't disagree but lets be consistent until we reindent the whole file, and use the same indentation as the rest of the code in that function.

Reply:
Ok, done.